include/Spectra/JDSymEigsBase.h

```diff
@@ -0,0 +1,185 @@
+// Copyright (C) 2020 Netherlands eScience Center <J.Wehner@esciencecenter.nl>
+//
+// This Source Code Form is subject to the terms of the Mozilla
+// Public License v. 2.0. If a copy of the MPL was not distributed
+// with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef SPECTRA_JD_SYM_EIGS_BASE_H
+#define SPECTRA_JD_SYM_EIGS_BASE_H
+
+#include <Eigen/Dense>
+#include <vector>     // std::vector
+#include <cmath>      // std::abs, std::pow
+#include <algorithm>  // std::min
+#include <stdexcept>  // std::invalid_argument
+#include <utility>    // std::move
+
+#include "Util/Version.h"
+#include "Util/TypeTraits.h"
+#include "Util/SelectionRule.h"
+#include "Util/CompInfo.h"
+#include "Util/SimpleRandom.h"
+#include "Util/SearchSpace.h"
+#include "Util/RitzPairs.h"
```

Comment:
do we need all these includes?

Reply:
I don't know, I just merge the `6_jd_base_solver` into DPR :)